content/en/docs/reference/resource-types/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Resource Types
+content_type: reference
+---
+
+<!-- overview -->
+
+This page provides information on the different resource types you can request, built into Kubernetes.
+
+<!-- body -->
+
+There are several different resource types: `cpu`, `memory`, `storage`, `hugepages-*`, and `emphemeral-storage`.
+
+
+Resource requests on Pods are an integral part of scheduling and 
+limiting and managing resource consumption in a Kubernetes 
+cluster. 
+See [Kubernetes Scheduler](/docs/concepts/scheduling-eviction/kube-scheduler/) and [Requests and Limits](/docs/concepts/configuration/manage-resources-containers/#requests-and-limits) for more details.
+## CPU 
```

Comment:
nit: Would you add a newline before the CPU heading? Thanks!

Reply:
Done!